### DISCUSSION TITLE
Relogin on entering a project [Mar 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5367] [Discord Threads]
> 🙋‍♂️ **Hirslay**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### can u please paste project info  
#  
> 💁 **o1lab**  
#### and more details on your setup  
#  
> 💁 **o1lab**  
#### proj info can be found in project menu  
#  
> 💁 **o1lab**  
#### on left top  
#  
> 🙋‍♂️ **Hirslay**  
#### I can't click any project. I can only edit project name or delete project (in actions)  
#  
> 🙋‍♂️ **Hirslay**  
#### When I'm trying to click any project it automaticly forces me to login again  
#  
> 👨‍💻 **pranavxc**  
#### Are you using docker or node  ?  
#  
> 🙋‍♂️ **Hirslay**  
#### Docker  
#  
> 🙋‍♂️ **Hirslay**  
#### I created (another) new project   
####    
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.98.3**  
#  
> 👨‍💻 **pranavxc**  
#### can you share the docker logs  
#  
> 🙋‍♂️ **Hirslay**  
#### not at this specific time sadly  
#  
> 👨‍💻 **pranavxc**  
#### Looks like you are using an older version, please upgrade to the latest version for more stability and features  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1075760829464588451)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
